compiler/dyno/lib/uast/chpl-syntax-printer.cpp

```diff
@@ -123,15 +123,85 @@ static std::string pragmaFlagsToString(const Decl* node) {
   return ret;
 }
 
-// TODO: Attributes
-// TODO: Nesting
-// TODO: Semicolons
-// TODO: Newlines
-// TODO: Parentheses based on operator precedence
+// stolen from convert-uast.cpp to prevent printing { } inside arrays
+// probably a better way to do this here
+static bool isBracketLoopMaybeArrayType(const uast::BracketLoop* node) {
```

Comment:
Can we make this a method on BracketLoop and use it from both places?

Reply:
I'll make a TODO for that in a later PR